src/posix/platform/netif.cpp

```diff
@@ -150,6 +150,15 @@ extern int
 unsigned int gNetifIndex = 0;
 char         gNetifName[IFNAMSIZ];
 
+const char *otSysGetThreadNetifName(void)
+{
+    return gNetifName;
+}
+
+unsigned int otSysGetThreadNetifIndex(void)
+{
+    return gNetifIndex;
+}
```

Comment:
1. nit: move to bottom to colocate with `otPlatGetNetif()`?
2. Is `otPlatGetNetif()` something we should keep?

Reply:
I forgot to delete the function. And I intentionally put it here so that it always available(even OPENTHREAD_CONFIG_PLATFORM_NETIF_ENABLE is not enabled).